sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -25,6 +25,658 @@ python -m pip install azure-ai-documentintelligence
+
+document_intelligence_client = DocumentIntelligenceClient(
+    endpoint=endpoint, credential=AzureKeyCredential(key)
+)
+with open(path_to_sample_documents, "rb") as f:
+    poller = document_intelligence_client.begin_analyze_document(
+        "prebuilt-layout", analyze_request=f, content_type="application/octet-stream"
+    )
+result = poller.result()
+
+for idx, style in enumerate(result.styles):
+    print(
+        "Document contains {} content".format(
+            "handwritten" if style.is_handwritten else "no handwritten"
+        )
+    )
+
+for page in result.pages:
+    print("----Analyzing layout from page #{}----".format(page.page_number))
+    print(
+        "Page has width: {} and height: {}, measured with unit: {}".format(
+            page.width, page.height, page.unit
+        )
+    )
+
+    for line_idx, line in enumerate(page.lines):
+        words = line.get_words()
+        print(
+            "...Line # {} has word count {} and text '{}' within bounding polygon '{}'".format(
+                line_idx,
+                len(words),
+                line.content,
+                line.polygon,
+            )
+        )
+
+        for word in words:
+            print(
+                "......Word '{}' has a confidence of {}".format(
+                    word.content, word.confidence
+                )
+            )
+
+    for selection_mark in page.selection_marks:
+        print(
+            "...Selection mark is '{}' within bounding polygon '{}' and has a confidence of {}".format(
+                selection_mark.state,
+                selection_mark.polygon,
+                selection_mark.confidence,
+            )
+        )
+
+for table_idx, table in enumerate(result.tables):
+    print(
+        "Table # {} has {} rows and {} columns".format(
+            table_idx, table.row_count, table.column_count
+        )
+    )
+    for region in table.bounding_regions:
+        print(
+            "Table # {} location on page: {} is {}".format(
+                table_idx,
+                region.page_number,
+                region.polygon,
+            )
+        )
+    for cell in table.cells:
+        print(
+            "...Cell[{}][{}] has content '{}'".format(
+                cell.row_index,
+                cell.column_index,
+                cell.content,
+            )
+        )
+        for region in cell.bounding_regions:
+            print(
+                "...content on page {} is within bounding polygon '{}'".format(
+                    region.page_number,
+                    region.polygon,
+                )
+            )
+
+print("----------------------------------------")
+```
+
+### Using the General Document Model
+
+Analyze key-value pairs, tables, styles, and selection marks from documents using the general document model provided by the Document Intelligence service.
+Select the General Document Model by passing `model_id="prebuilt-document"` into the `begin_analyze_document` method:
```

Comment:
Wasnt this model deprecated in the current service API version?

Reply:
I can confirm it was deprecated. We get a 404 if we try to use it with the new version.

Reply:
@YalinLi0312 we shouldnt be recommending a deprecated model to users in the README

Reply:
Removed. `prebuilt-document` and `prebuilt-businessCard` are deprecated.